src/main/java/uk/ac/ebi/ampt2d/accession/sample/SampleModelSummaryFunction.java

```diff
@@ -0,0 +1,30 @@
+/*
+ *
+ * Copyright 2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package uk.ac.ebi.ampt2d.accession.sample;
+
+import java.util.stream.Collectors;
+
+public class SampleModelSummaryFunction implements java.util.function.Function<SampleModel, String> {
+
+    @Override
+    public String apply(SampleModel message) {
+        return message.getSampleProperties().values().stream().sorted().collect(Collectors.joining(","));
```

Comment:
Two different samples could have the sample summary. Imagine we have the same values in two samples but in different properties:

- Sample 1
  - Property a: S1
  - Propery b: M
  - Property c: S2

- Sample 2
  - Property a: S2
  - Property b: M
  - Property c: S1

The summary for both samples is the same: M,S1,S2

Reply:
It is a map because it was not really defined. It won't be a map when it is properly defined.

Reply:
Then a TODO would be helpful here and in the Study analogous class